frontend/src/components/LandingPage/Explanation/Explanation.css

```diff
@@ -0,0 +1,25 @@
+
+.title {
+    font-size: 100px;
+    color: #c6bbb9;
+    text-align: center;
+    letter-spacing: 6px;
+    
+}
+
+.Jumbo1{
+    /*background-image: url("./galaxy-2.jpg");*/
```

Comment:
Are we using the image as a background? If not what do you think about removing it?